setup/cocci.m4

```diff
@@ -285,7 +285,7 @@ AC_DEFUN([AC_COCCI_TOOL],
     AC_SUBST([$1], [no])
   ],
   [dnl  find the tool
-    AC_COCCI_FINDTOOL([$1],[[$]$1])
+    AC_COCCI_FINDTOOL([$1],[$2])
```

Comment:
Does your suggestion mean that the specification "`[$]`" should eventually be changed also at other places in this script?